src/omnisharp/OmnisharpManager.ts

```diff
@@ -22,10 +22,10 @@ export class OmnisharpManager {
         private platformInfo: PlatformInformation) {
     }
 
-    public async GetOmniSharpLaunchInfo(defaultOmnisharpVersion: string, omnisharpPath: string, useFramework: boolean, serverUrl: string, latestVersionFileServerPath: string, installPath: string, extensionPath: string): Promise<LaunchInfo> {
-        if (!omnisharpPath) {
+    public async GetOmniSharpLaunchInfo(defaultOmnisharpVersion: string, omnisharpPath: string | undefined, useFramework: boolean, serverUrl: string, latestVersionFileServerPath: string, installPath: string, extensionPath: string): Promise<LaunchInfo> {
+        if (omnisharpPath === undefined) {
```

Comment:
```suggestion
        if (omnisharpPath === undefined || omnisharpPath.length === 0) {
```

Reply:
Good catch, you're right. The integration tests also exposed a related configuration problem: the defaults for string-related options are all set to `null` at the moment. Going to push up a test commit so I can run the tests using a Codespace that actually has .NET installed.

Reply:
Right, it's because VS Code coerces undefined values for known settings into preset default values: https://github.com/microsoft/vscode/blob/6f26fa19db7e4cd2af57f52fd7009e715c3d7aab/src/vs/platform/configuration/common/configurationRegistry.ts#L578

Of course, this isn't documented anywhere I could find in their API documentation. :)